towhee/serve/triton/bls/pipeline_model.py

```diff
@@ -27,7 +27,7 @@ def auto_complete_config(auto_complete_model_config):
         input0 = {'name': 'INPUT0', 'data_type': 'TYPE_STRING', 'dims': [1]}
         output0 = {'name': 'OUTPUT0', 'data_type': 'TYPE_STRING', 'dims': [1]}
 
-        auto_complete_model_config.set_max_batch_size(0)
+        auto_complete_model_config.set_max_batch_size(8)
```

Comment:
why is the magic number 8?

Reply:
Can achieve a better performance in image-pipeline, and will be configured in the next version.